### PR TITLE
Add doc/ref examples for bucket policy/notification, listObjectV2, removeIncompleteUpload

### LINKED ISSUE #506: doc: reference missing bucket notification examples
https://github.com/minio/minio-js/blob/master/README.md

### PR DESCRIPTION
**Pull request by vadmeste**
Fixes #506 #507 #509 #510

### CODE REVIEW

**Comment by vadmeste** [comment]
> Move this back to object operations

you mean for 'removeIncompleteUpload' ?

**Comment by harshavardhana** [comment]
> you mean for 'removeIncompleteUpload' ?

yes

File: README.md [hunk 1]
```diff
@@ -180,13 +193,22 @@ The full API Reference is available here.
...
 
 #### Full Examples : Presigned Operations
 * [presigned-getobject.js](https://github.com/minio/minio-js/blob/master/examples/presigned-getobject.js)
 * [presigned-putobject.js](https://github.com/minio/minio-js/blob/master/examples/presigned-putobject.js)
 * [presigned-postpolicy.js](https://github.com/minio/minio-js/blob/master/examples/presigned-postpolicy.js)
 
+#### Full Examples: Bucket Notification Operations
+* [get-bucket-notification.js](https://github.com/minio/minio-js/blob/master/examples/get-bucket-notification.js)
+* [set-bucket-notification.js](https://github.com/minio/minio-js/blob/master/examples/set-bucket-notification.js)
+* [remove-all-bucket-notification.js](https://github.com/minio/minio-js/blob/master/examples/remove-all-bucket-notification.js)
```
Comment (harshavardhana) [inline README.md]: listen bucket notification is missing.

Comment (harshavardhana) [inline README.md]: https://github.com/minio/minio-go#api-reference--bucket-notification-operations - we need to mention it like (Minio Extension)

**Pull request merged**